files/dhcp/dhclient.conf

```diff
@@ -19,7 +19,7 @@ option acl-url code 226 = text;
 send host-name = gethostname();
 request subnet-mask, broadcast-address, time-offset, routers,
         domain-name, domain-name-servers, domain-search, host-name,
-        dhcp6.name-servers, dhcp6.domain-search,
-        netbios-name-servers, netbios-scope, interface-mtu,
-        rfc3442-classless-static-routes, ntp-servers, snmp-community, minigraph-url;
+        dhcp6.name-servers, dhcp6.domain-search, interface-mtu,
```

Comment:
For information, the current value of option 55 in DHCPRequest is:
            Parameter-Request Option 55, length 14: 
              Subnet-Mask, BR, Time-Zone, Default-Gateway
              Domain-Name, Domain-Name-Server, Option 119, Hostname
              MTU, Classless-Static-Route, NTP, Option 224
              Option 225, Option 226